src/components/menu.js

```diff
@@ -9,10 +9,10 @@ import menubg from '../assets/menuBg.svg';
 import { Link } from 'react-router-dom';
 import { withNamespaces } from 'react-i18next';
 import { Context } from '../Context';
-import { useContext } from 'react';
 function Menu(props) {
   const [state] = useContext(Context);
   function handleLogOut() {}
+  const [setLang] = useState('EN');
```

Comment:
this won't work because the first element of the array isn't the setter, so you're putting the state inside setLang.

Just delete the line and the lines that are using it below